package.json

```
@@ -27,21 +27,36 @@
     "katex": "^0.16.22",
     "lucide-react": "^0.503.0",
-    "next": "^15.3.2",
+    "monaco-editor": "^0.52.2",
+    "next": "15.2.0",
     "next-auth": "^5.0.0-beta.28",
     "next-themes": "^0.4.6",
     "react": "^19.0.0",
     "react-dom": "^19.0.0",
     "react-katex": "^3.1.0",
-    "tailwind-merge": "^3.2.0"
+    "tailwind-merge": "^3.2.0",
```

Biome (1.9.4)

[error] The key tailwind-merge was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)